src/parser.rs

```diff
@@ -73,8 +93,18 @@ pub fn parse_and_generate_code(schema_file_contents: String, config: &Generation
         }
     }
 
+    let mut tables_hm = std::collections::HashMap::new();
+
     for table in tables.iter_mut() {
-        table.generated_code = code::generate_table(table.clone(), config);
+        // table.generated_code = code::generate_table(table.clone(), config);
+        tables_hm.insert(table.name.clone(), table);
+    }
+
+    for table in tables_hm.iter_mut() {
+        // **table.1 = code::generate_table(table.clone(), config, &tables_hm);
+
+        // (table.1).generated_code = String::new();
+        table.1.generated_code = String::new();
```

Comment:
this looks like a bit more than just a fmt change, will this modify the behavior of the program?

Reply:
That's awkward! This shouldn't be here, my mistake!
Thanks for the review by the way! Gonna send the fixup for this!

Reply:
Yeah, no worries.

Looks better on my activity graph if I'm contributing in more ways than just commits, lol.